### ISSUE TITLE
Compatibility with python 3.5

### ISSUE THREAD

**Issue by Insoleet** [ucoin-io/cutecoin#241]
```
ERROR:main:async_exception_handler:Task exception was never retrieved
future: <Task finished coro=<refresh_block() done, defined at /home/inso/code/ucoin/cutecoin/src/cutecoin/core/net/node.py:284> exception=AttributeError("'_DetachedRequestContextManager' object has no attribute 'cr_code'",)>
Traceback (most recent call last):
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 270, in _format_coroutine
    coro_code = coro.gi_code
AttributeError: '_DetachedRequestContextManager' object has no attribute 'gi_code'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.5/asyncio/tasks.py", line 239, in _step
    result = coro.send(value)
  File "/home/inso/code/ucoin/cutecoin/src/cutecoin/core/net/node.py", line 294, in refresh_block
    block_data = yield from bma.blockchain.Current(conn_handler).get()
  File "/usr/lib/python3.5/site-packages/ucoinpy/api/bma/blockchain/__init__.py", line 87, in __get__
    r = yield from self.requests_get('/current', **kwargs)
  File "/usr/lib/python3.5/site-packages/ucoinpy/api/bma/__init__.py", line 109, in requests_get
    headers=self.headers), 15)
  File "/usr/lib/python3.5/asyncio/tasks.py", line 365, in wait_for
    fut = ensure_future(fut, loop=loop)
  File "/usr/lib/python3.5/asyncio/tasks.py", line 526, in ensure_future
    task = loop.create_task(coro_or_future)
  File "/usr/lib/python3.5/asyncio/base_events.py", line 216, in create_task
    task = tasks.Task(coro, loop=self)
  File "/usr/lib/python3.5/asyncio/tasks.py", line 77, in __init__
    self._loop.call_soon(self._step)
  File "/usr/lib/python3.5/site-packages/quamash/__init__.py", line 370, in call_soon
    return self.call_later(0, callback, *args)
  File "/usr/lib/python3.5/site-packages/quamash/__init__.py", line 345, in call_later
    .format(callback, args, delay))
  File "/usr/lib/python3.5/asyncio/futures.py", line 194, in __repr__
    info = self._repr_info()
  File "/usr/lib/python3.5/asyncio/tasks.py", line 102, in _repr_info
    coro = coroutines._format_coroutine(self._coro)
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 272, in _format_coroutine
    coro_code = coro.cr_code
AttributeError: '_DetachedRequestContextManager' object has no attribute 'cr_code'
```

**Comment by M5oul** [comment]
https://github.com/harvimt/quamash/issues/50
Seems to be an `aiohttp` bug with `asyncio.wait_for`.

**Comment by Insoleet** [comment]
Fixed with  aiohttp 0.18.3
